source/js/navigation.js

```diff
@@ -11,6 +11,7 @@ $(document).ready(function () {
     const searchBtnExitImgSrc = getCloseImgUrl(searchBtnImgSrc, 'close_black');
     const searchBtn = $("#search");
     const langBtn = $("#langBtn");
+    const dropdowns = document.querySelectorAll("li div.sub-menu-btn");
```

Comment:
hmmm suddenly we have quote mark inconsistency again - how come this is not picked up by the linter?